homeassistant/helpers/dispatcher.py

```diff
@@ -225,4 +225,4 @@ def async_dispatcher_send(
         if job is None:
             job = _generate_job(signal, target)
             target_list[target] = job
-        hass.async_run_hass_job(job, *args)
+        hass.async_run_hass_job(job, *args, eager_start=True)
```

Comment:
Do we still need to allow passing `eager_start=True` to `async_run_hass_job`? Why not always do it? The idea of `async_run_hass_job` is that it needs to run asap.

Reply:
https://github.com/home-assistant/core/pull/111683 aimed to do that but I realized I had to do it in pieces because it was too much many tests to fix in one PR

Reply:
Once we have everything converted the plan is to drop the kwarg from the run job variants